platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -698,16 +684,21 @@ public void bind(
         notifierWiring.bind(notifier);
         platformPublisherWiring.bind(platformPublisher);
         stateGarbageCollectorWiring.bind(builder::buildStateGarbageCollector);
+        gossipWiring.bind(builder.buildGossip());
     }
 
     /**
-     * Get the input wire gossip. All events received from peers during should be passed to this wire.
-     *
-     * @return the wire where all events from gossip should be passed
+     * Start gossiping.
      */
-    @NonNull
-    public InputWire<GossipEvent> getGossipEventInput() {
-        return gossipWiring.eventInput();
+    public void startGossip() {
+        gossipWiring.getStartInput().put(NoInput.getInstance());
```

Comment:
any reason not to inject here? (also for the stop)

Reply:
No reason. Changed as suggested.